### PR TITLE
global: refactor keywords to tags

### LINKED ISSUE #479: rename keywords to tags
- [x] in all the code (record type and all the references, also UI display)
- [x] add keywords field to the document as a free text field

### PR DESCRIPTION
**Pull request by topless**
closes #479

### CODE REVIEW

File: invenio_app_ils/schemas/documents/document-v1.0.0.json [hunk 1]
```diff
@@ -513,14 +513,14 @@
...
-    "keyword_pids": {
+    "tag_pids": {
       "type": "array",
       "items": {
         "type": "string"
       },
       "uniqueItems": true
     },
-    "keywords": {
+    "tagss": {
```
Comment (kpsherva) [inline invenio_app_ils/schemas/documents/document-v1.0.0.json]: 👀 double `s`

File: invenio_app_ils/ui/public/index.html [hunk 2]
```diff
@@ -27,7 +27,10 @@
...
+    <div
```
Comment (kpsherva) [inline invenio_app_ils/ui/public/index.html]: the config is unescaped for some reason

**Review by ntarocco — DISMISSED** [review]

**Comment by ntarocco** [comment]
@topless you are missing the second part of the task. The `keywords` field is a simple text field that should be added to the document schema + mapping

**Comment by ntarocco** [comment]
@topless can you please change the commit text and reference the right issue?
it is the #479 thanks! (while here I see the 491)

**Comment by topless** [comment]
> @topless you are missing the second part of the task. The `keywords` field is a simple text field that should be added to the document schema + mapping

@ntarocco just to clarify, the keywords used to be and still are `$refs` in the document schema. The only change I did was this, https://github.com/inveniosoftware/invenio-app-ils/blob/b2328a32c6389c0a9ce82679fa8dafd4d78d473c/invenio_app_ils/mappings/v6/documents/document-v1.0.0.json#L107 is it good enough? Or you imply a complete restructure to the way keywords work?

File: invenio_app_ils/ui/public/index.html [hunk 3]
```diff
@@ -27,7 +27,7 @@
...
...tChildRelation", "schema": "invenio_pidrelations.serializers.schemas.RelationSchema"}, {"api": "invenio_app_ils.records.relations.nodes:PIDNodeRelated", "id": 0, "label": "Language", "name": "language", "relation_class": "invenio_app_ils.records.relations.api.SiblingsRelation", "schema": "invenio_pidrelations.serializers.schemas.RelationSchema"}, {"api": "invenio_app_ils.records.relations.nodes:PIDNodeRelated", "id": 1, "label": "Edition", "name": "edition", "relation_class": "invenio_app_ils.records.relations.api.SiblingsRelation", "schema": "invenio_pidrelations.serializers.schemas.RelationSchema"}, {"api": "invenio_app_ils.records.relations.nodes:PIDNodeRelated", "id": 2, "label": "Other", "name": "other", "relation_class": "invenio_app_ils.records.relations.api.SiblingsRelation", "schema": "invenio_pidrelations.serializers.schemas.RelationSchema"}], "rest_mimetype_query_arg_name": "format", "series": {"search": {"aggs": ["moi"], "sortBy": {"onEmptyQuery": "mostrecent", "values": [{"field": "mostrecent", "order": 1, "title": "Newest"}, {"field": "bestmatch", "order": 2, "title": "Best match"}]}, "sortOrder": ["asc", "desc"]}}, "support_email": "info@inveniosoftware.org"}'></div>
```
Comment (ntarocco) [inline invenio_app_ils/ui/public/index.html:30]: I think should be encoded no?

**Pull request merged**